weblate/checks/chars.py

```diff
@@ -327,10 +327,10 @@ class EscapedNewlineCountingCheck(CountingCheck):
 
 
 class NewLineCountCheck(CountingCheck):
-    r"""Check whether there is same amount of new lines"""
+    r"""Check whether the amount of newlines matches"""
     string = '\n'
     check_id = 'newline-count'
-    name = _('Newline charater count mismatch')
+    name = _('Mismatching newline character count')
```

Comment:
```suggestion
    name = _('Mismatching line breaks')
```
I think this (typographic) variant makes sense over "Mismatching number of new lines".
"Added line breaks" does away with the idea of wrapping, but alas.